cmake/OpenCVGenRV64V.cmake

```diff
@@ -0,0 +1,9 @@
+set(CMAKE_SYSTEM_NAME "Linux")
```

Comment:
This is wrong place for toolchain file.

Linux-based cross-compilation toolchains are stored under `platforms/linux`.

Reply:
The toolchain  is here "https://github.com/damonyu1989/Tools"